src/features/app/context/guestLocale.js

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useState, useEffect } from 'react';
+
+export const GuestLocaleContext = createContext();
+
+const STORAGE_KEY = 'RAB_GUEST_LOCALE';
+const DEFAULT_GUEST_LOCALE = 'en';
+
+// TODO: Util?
+const persistState = (state) => {
+  window.localStorage.setItem(STORAGE_KEY, JSON.stringify(state));
```

Comment:
you don't need to use `window.`, you can just use `localStorage.setItem`. and the JSON.parse/JSON.stringify is not necessary as it is only a string, right?

Reply:
with that you can get rid of this helper (that becomes an even simpler wrapper). and maybe even get rid of the function to initialize the state